jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     '!**/node_modules/**',
   ],
   setupFilesAfterEnv: ['<rootDir>/setupTests.js'],
-  testPathIgnorePatterns: ['/node_modules/', '/.next/', '/out'],
+  testPathIgnorePatterns: ['/node_modules/', '/.next/', '/out', 'cypress'],
```

Comment:
```suggestion
  testPathIgnorePatterns: ['/node_modules/', '/.next/', '/out', '/cypress'],
```

If the others are using a leading slash...